utils/utest.yaml

```diff
@@ -2,8 +2,10 @@
 #
 #  SPDX-License-Identifier: BSD-2-Clause-Patent
 - name: include checks
+  memcheck: False
+  gha: True
   tests:
-    - cmd: ["utils/include_test.py"]
+    - cmd: ["utils/unit/include_test.py"]
```

Comment:
I presume the idea is to eventually run more of them?

Reply:
Maybe this could be more generic...such as 
tags: gha

and using --tags gha

Reply:
@daltonbohning could comment on this.  There already is a `--suite` option but it does string matching rather than using tags so it's not much use IMO.

Reply:
I recently added this one, which might belong here ( I need to think about it some more)
https://github.com/daos-stack/daos/blob/40f9ffaa9dbc97f68be6450cdf9ec58c371c6c1e/utils/utest.yaml#L185-L189
And I suspect there are more we could group together.

And at least for python tests, we might consider running `pytest` against a directory of unit tests.

Reply:
Sorry, I didn't mean you had to group those here, but that works! :)

Reply:
I did it mostly to see what the output looks like.  actions only appears to have one level of grouping.